source/includes/debt-settlement.md.erb

```diff
@@ -516,25 +521,94 @@ Content-Type: application/json
 ### Description
 Adds a repayment to an investment.
 
+<aside class="warning">
+Any users of this endpoint who are not registering a reference first will need to move to <a href="#settlement-debt-post-platformapi-settlement-investor-repayment">the new behaviour</a>.
+</aside>
+
  - The system will record the repayments and wait for a deposit into your receiving account matching the amount and reference you specified
 when making the API call.
  - Once the money is deposited the system will automatically distribute the funds to investors and notify them of a repayment.
  - The system will publish a <a href="/#webhooks-batch_update">BATCH_UPDATE</a> webhook with a type of `REPAYMENTS` to update you on the progress of the distribution.
 
+### Request
+| Name                              | Type   | Description                                                                    | Required |
+| --------------------------------- | ------ | ------------------------------------------------------------------------------ | -------- |
+| reference                         | string | The payment reference used for the repayment of funds. Will also be present on `INVESTOR_FUNDS_RECEIVED` webhook | required |
+| investorRepayments                | array  | The repayments per investor.                                                   | required |
+| investorRepayments[].investmentId | string | The ID of the of the investment                                                | required |
+| investorRepayments[].type         | string | The type of the repayment Possible values are: <br>`CAPITAL`<br>`INTEREST`<br> | required |
+| investorRepayments[].amount       | ref    | The amount being repaid                                                        | required |
+| investorRepayments[].tax          | ref    | The amount of tax being withheld from this repayment                           | optional |
+
+## `POST /platformApi/settlement/investor-repayment`
+
+```http
+
+POST /platformApi/settlement/investor-repayment HTTP/1.1
+Host: api-sandbox.goji.investments
+Content-Type: application/json
+Authorization: Basic ...
+
+{
+  "reference" : "reference",
+  "sourceWalletId": "sourceWalletId",
+  "investorRepayments" : [ {
+    "amount" : {
+      "amount" : 2.68,
+      "currency" : "currency"
+    },
+    "investmentId" : "investmentId",
+    "tax" : {
+      "amount" : 2.68,
+      "currency" : "currency"
+    },
+    "type" : "CAPITAL"
+  }, {
+    "amount" : {
+      "amount" : 2.68,
+      "currency" : "currency"
+    },
+    "investmentId" : "investmentId",
+    "tax" : {
+      "amount" : 2.68,
+      "currency" : "currency"
+    },
+    "type" : "CAPITAL"
+  } ]
+}
+```
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+""
```

Comment:
I thought we were returning the repayment ID now?

Reply:
Yeah my bad. Done 👍